src/components/AtomicOrdersTable/AtomicOrdersTable.js

```diff
@@ -10,10 +14,15 @@ import './style.css'
 
 const AtomicOrdersTable = ({
   atomicOrders, filteredAtomicOrders, renderedInTradingState,
-  cancelOrder, authToken, gaCancelOrder, getMarketPair, editOrder,
+  cancelOrder, authToken, gaCancelOrder, getMarketPair, editOrder, markets,
 }) => {
   const [ref, size] = useSize()
-  const data = renderedInTradingState ? filteredAtomicOrders : atomicOrders
+  const data = useMemo(() => {
+    const orders = renderedInTradingState ? filteredAtomicOrders : atomicOrders
+    const filtered = _filter(_keys(orders), (key) => !_isUndefined(markets[orders[key]?.symbol]))
+    return _reduce(filtered, (res, key) => ({ ...res, [key]: orders[key] }), {})
+  }, [renderedInTradingState, filteredAtomicOrders, atomicOrders, markets])
+
```

Comment:
let's add this logic to getAtomicOrders selector, so it automatically works for count and filtered/all atomic orders. so we can avoid repetition of logic in getFilteredAtomicOrdersCount selector.